<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPolygonLabel</class>
 <widget class="QWidget" name="QPolygonLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Polygon 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_outer_color">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>15</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_inner_color">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>45</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
